### PR TITLE
First pass at an embedded player

### PR DESCRIPTION
**Pull request by johnlindquist**
The `_app.js` check for an `/embed` at the end of the route and removes all layout/chrome if it exists.

This currently only matches a `/lessons/[slug]/embed.txt`, but maybe there are other scenarios where people would want to iframe some content from egghead? 🤔

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -25,6 +25,12 @@ Router.events.on('routeChangeComplete', () => window.ahoy.trackView())
 export default class App extends NextApp {
   render() {
     const {Component, pageProps} = this.props
+    const {pathname} = this.props.router
+
+    if (pathname.endsWith(`/embed`)) {
+      return <Component {...pageProps}></Component>
```
Comment (johnlindquist) [inline src/pages/_app.tsx:31]: Removing all wrapping components.

We probably still want some Head/Metadata/Tracking?

File: src/pages/lessons/[slug]/embed.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
+/** @jsx jsx */
```
Comment (johnlindquist) [inline src/pages/lessons/[slug]/embed.tsx:1]: I copy/pasted the lessons page and removed non-video player stuff

**Comment by joelhooks** [comment]
the egghead embed player introduces a couple of things. Mainly it's using PReact and has several essential query params. I'd probably be inclined to deprecate it entirely tbh and just have folks link to lessons.

**Comment by joelhooks** [comment]
I'm gonna close this for now.

**Pull request closed without merging**